packages/tokens/src/tokens/breakpoint.ts

```diff
@@ -0,0 +1,7 @@
+export const breakpoint = {
+  sm: "600px",
```

Comment:
One thing I meant to ask: where are these breakpoints coming from? Did we grab them from another framework defaults? Or is this our own definition?

Reply:
Didn't think to pull from a framework. I've just used these breakpoints since the dawn of time. Maybe it makes sense to grab the ones defined here by tailwind instead? https://tailwindcss.com/docs/screens

Reply:
> Maybe it makes sense to grab the ones defined here by tailwind instead? https://tailwindcss.com/docs/screens

Yeah that seems like a pretty sensible choice / starting place? And document where that came from for completeness.